arcaflow_plugin_uperf/uperf_schema.py

```diff
@@ -131,6 +124,15 @@ class ProfileFlowOpConnection(ProfileFlowOpCommon):
             "description": "The SSL Engine used by OpenSSL",
         },
     )
+    port: int = field(
+        default=20001,
+        metadata={
+            "name": "port",
+            "description": "The port used to establish the connection used by "
+                           "the FlowOp. This is not the same as the comm_port.\n"
+                           "UPerf does not have any documentation on this.",
```

Comment:
This final sentence seems to be pretty much true, but is quite ... unsatisfying. I'm also not sure it really adds anything.

Reply:
It also might become wrong without our knowing.

Reply:
I removed it, and replaced it with updated documentation on the new option to set it to None.